test/Swashbuckle.AspNetCore.SwaggerGen.Test/Swashbuckle.AspNetCore.SwaggerGen.Test.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>1701;1702;1591</NoWarn>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Not 100% sure how this will reconcile with other changes in the backlog, should this multitarget and the tests that require .NET 7 or greater be `#ifdef`'d?

Reply:
Good question, I'm not sure. For now maybe `TargetFrameworks` would be the more conservative approach?